### ISSUE TITLE
Pass pathom env

### ISSUE THREAD

**Issue by mitchelkuijpers** [denisidoro/graffiti#31]
Hi I tried to use your library to expose our existing pathom API. One thing we ran into is that it is currently impossible to pass a pathom env. Is that something you would like to add? I have already hacked in the possibility to do this, so I could make a PR.

**Comment by denisidoro** [comment]
Yes, please! Could you do so in such a way that everything is configurable via graffiti/compile?

I didn't know other people were using this. I'll probably release it to clojars then
